src/modules/options/OptionsWidget_theme.cpp

```diff
@@ -44,12 +46,38 @@ OptionsWidget_theme::OptionsWidget_theme(QWidget * parent)
 	    KVI_OPTION_BOOL(KviOption_boolUseGlobalApplicationFont));
 	connect(b, SIGNAL(toggled(bool)), f, SLOT(setEnabled(bool)));
 
-	addRowSpacer(0, 3, 1, 3);
+	addLabel(0, 3, 0, 3, __tr2qs_ctx("Qt Style:", "options"));
+	m_pQtStyle = new QComboBox(this);
+	addWidgetToLayout(m_pQtStyle, 1, 3, 1, 3);
+	m_pQtStyle->addItem(DEFAULT_QT_STYLE);
+	foreach(QString key, QStyleFactory::keys()) {
```

Comment:
From Qt docs: New code should prefer C++11 range-based loops.
```suggestion
	for (const QString& key : QStyleFactory::keys()) {
```